src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -41,7 +41,7 @@ public sealed class DocumentConventions : Client.Conventions
         public delegate bool TryConvertValueToObjectForQueryDelegate<in T>(string fieldName, T value, bool forRange, out object objValue);
 
 #if NETCOREAPP
-        internal static HttpVersionPolicy? DefaultHttpVersionPolicy;
+        internal static HttpVersionPolicy? DefaultHttpVersionPolicy = System.Net.Http.HttpVersionPolicy.RequestVersionOrHigher;
```

Comment:
this will also change the policy for the Client API, which we want to avoid

Reply:
removed.